integration-tests/ws-security-policy/src/main/resources/saml1-policy.xml

```diff
@@ -5,33 +5,11 @@
         xmlns:sp="http://docs.oasis-open.org/ws-sx/ws-securitypolicy/200702">
     <wsp:ExactlyOne>
         <wsp:All>
-            <sp:TransportBinding>
-                <wsp:Policy>
-                    <sp:TransportToken>
-                        <wsp:Policy>
-                            <sp:HttpsToken>
-                                <wsp:Policy/>
-                            </sp:HttpsToken>
-                        </wsp:Policy>
-                    </sp:TransportToken>
-                    <sp:Layout>
-                        <wsp:Policy>
-                            <sp:Lax/>
-                        </wsp:Policy>
-                    </sp:Layout>
-                    <sp:IncludeTimestamp/>
-                    <sp:AlgorithmSuite>
-                        <wsp:Policy>
-                            <sp:Basic128/>
-                        </wsp:Policy>
-                    </sp:AlgorithmSuite>
-                </wsp:Policy>
-            </sp:TransportBinding>
             <sp:AsymmetricBinding>
                 <wsp:Policy>
                     <sp:InitiatorToken>
                         <wsp:Policy>
-                            <sp:X509Token sp:IncludeToken="http://docs.oasis-open.org/ws-sx/ws-securitypolicy/200702/IncludeToken/Always">
+                            <sp:X509Token sp:IncludeToken="http://docs.oasis-open.org/ws-sx/ws-securitypolicy/200702/IncludeToken/AlwaysToRecipient">
                                 <wsp:Policy>
                                     <sp:WssX509V3Token10/>
                                     <sp:RequireEmbeddedTokenReference/>
```

Comment:
The policy xml will be used both at client and server side. And InitiatorToken only sign the request message from client to server, so we should be more specific that this token is included only for the request message from client to server.